### ISSUE TITLE
24 hrs

### ISSUE THREAD

**Issue by Cooper81** [D34DC3N73R/ha-nfl-card#8]
First of all, I would like to thank you for the great card. I use it with several teams and I love it.

But now I have a question: would it be possible to display the next kick-off time in 24-hour format?

Thank you in advance.

**Comment by D34DC3N73R** [comment]
yeah, that's probably possible. To be clear, you're not talking about the "Kickoff in" line, but the time displayed in the center of the Pregame version of the card, correct?

**Comment by Cooper81** [comment]
yes, im talking about the time in the middle:

![image](https://github.com/user-attachments/assets/6e21ac58-ac27-4a1b-ae5a-582d82db12a9)

Or maybe you can change the language in the card

**Comment by D34DC3N73R** [comment]
Thanks for the screenshot. It looks like you're using the [Team Tracker Card](https://github.com/vasqued2/ha-teamtracker-card), which is a fork of this card that was designed just for the NFL. It would be better to request that feature in that repo instead.

**Comment by Cooper81** [comment]
actually I use the nfl card

![image](https://github.com/user-attachments/assets/cf963c48-f173-4ac1-9655-5d424c1cee6e)

I'm trying the team tracker card.
many thanks

**Comment by D34DC3N73R** [comment]
Let me know if it doesn't work out and I can make the change here.
